curvefs/src/client/rpcclient/mds_client.cpp

```diff
@@ -214,6 +214,28 @@ FSStatusCode MdsClientImpl::GetFsInfo(uint32_t fsId, FsInfo *fsInfo) {
     return ReturnError(rpcexcutor_.DoRPCTask(task, mdsOpt_.mdsMaxRetryMS));
 }
 
+bool MdsClientImpl::CommitTx(const std::vector<PartitionTxId>& txIds) {
+    auto task = RPCTask {
+        CommitTxResponse response;
+        mdsbasecli_->CommitTx(txIds, &response, cntl, channel);
+        if (cntl->Failed()) {
+            LOG(WARNING) << "CommitTx Failed, errorCode = " << cntl->ErrorCode()
+                         << ", errorText =" << cntl->ErrorText()
+                         << ", logId = " << cntl->log_id();
+            return -cntl->ErrorCode();
+        }
+
+        auto rc = response.statuscode();
+        if (rc != curvefs::mds::topology::TopoStatusCode::TOPO_OK) {
+            LOG(WARNING) << "CommitTx: retCode = " << rc
+                         << ", message = " << TopoStatusCode_Name(rc);
+            return 1;
```

Comment:
do not return 0 or 1 here.

There are two types of error codes for RPCTask, <0 means rpc error, >=0(FSStatusCode) means logic error, rpcexcutor_ will to do retry differently according to the error code

Reply:
> do not return 0 or 1 here.
> 
> There are two types of error codes for RPCTask, <0 means rpc error, >=0(FSStatusCode) means logic error, rpcexcutor_ will to do retry differently according to the error code

yep, it's my oversight, i will improve it.